src/test/java/bio/terra/service/resourcemanagement/OneProjectPerProfileIdSelectorTest.java

```diff
@@ -1,11 +1,15 @@
 package bio.terra.service.resourcemanagement;
 
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
I'm interested in the reformatting of imports on this PR. Are they from spotless or some other formatter? Or from IntelliJ itself? I think it's ok to reformat them as long as the next person doesn't need to format them back. We don't want to have churn in the code base due to dueling formatters.